src/download.rs

```diff
@@ -242,13 +245,26 @@ pub fn download_file(
     match pb {
         Some(ref pb) => {
             copy_largebuf(&mut pb.wrap_read(response), &mut buffer)?;
-            pb.finish_with_message("Done");
+            pb.finish();
         }
         None => {
             copy_largebuf(&mut response, &mut buffer)?;
         }
     }
 
+    let downloaded_length: u64;
+    if let Some(total_length) = total_length {
+        downloaded_length = total_length - starting_length;
+    } else {
+        downloaded_length = fs::metadata(dest_name).unwrap().len();
+    }
+    eprintln!(
+        "Done. {} in {} ({}/s)",
+        HumanBytes(downloaded_length),
+        FormattedDuration(starting_time.elapsed()),
```

Comment:
Would [`HumanDuration`](https://docs.rs/indicatif/0.15.0/indicatif/struct.HumanDuration.html) instead of `FormattedDuration` be better?

Reply:
This is to print seconds even for more than a minute.

Reply:
I think `HumanDuration` is ok because if you want to know the exact time it's still next to the progress bar. `00:00:03` for "3 seconds" is a bit ugly. But you're right that it is unfortunate that 1:20 would just become `1 minute`.

@ducaale, what do you think?

Reply:
It is possible to write a[ custom function](https://www.reddit.com/r/rust/comments/gju305/getting_chronos_duration_to_format_to_hhmmss/fqo9zbb?utm_source=share&utm_medium=web2x&context=3) for outputting human-readable duration. But what should human-readable duration look like?

* `1min 60s` or `1 minute 60 seconds`?
* should it use `and` for separating the minute and seconds? what about hours?

> if you want to know the exact time it's still next to the progress bar

I was hoping that we could replace `elapsed_time` with `percent` so it can look like something like this:

<img width="806" alt="Screenshot 2021-04-02 220047" src="https://user-images.githubusercontent.com/16341131/113453966-084ade80-93ff-11eb-8fcb-a1050285efc2.png">

Reply:
@ducaale Changed the time taken to human-readable (Use [Human Time](https://crates.io/crates/humantime)).